airflow-core/src/airflow/ui/src/layouts/Nav/AdminButton.tsx

```diff
@@ -52,6 +53,7 @@ const links = [
 ];
 
 export const AdminButton = ({ authorizedMenuItems }: { readonly authorizedMenuItems: Array<MenuItem> }) => {
+  const { t: translate } = useTranslation("common");
```

Comment:
```suggestion
  const { t: translate } = useTranslation();
```

common is the default, no? so we don't need to specify it?

Reply:
Right, 'common' is the default namespace. I was just thinking, perhaps keeping 'common' explicit would be a bit clearer for other developers when they're working with the translations?